app/graphql/types/query_type.rb

```diff
@@ -40,5 +40,16 @@ def get_user(id:)
     def get_pairings(user_id:)
       Pairing.where(user1_id: user_id).or(Pairing.where(user2_id: user_id))
     end
+
+    # ----- topics and translations -----
+    field :get_topic_and_translations, Types::TopicType, null: false, description: 'Returns a random topic and any requested translations' do
+      argument :language_ids, [ID], required: true
+    end
+
+    def get_topic_and_translations(language_ids:)
+      raise GraphQL::ExecutionError, 'languageIds must contain two languages' if language_ids.size != 2
+      context[:language_ids] = language_ids
+      topic = Topic.random
```

Comment:
Is context basically saying given these two ids, return me a random topic with the translations for these two languages? I'm having trouble understanding because it's not very explicit that any translation is being returned

Reply:
Context is allowing me to access those params (language_ids) elsewhere outside of this query, which was being used in `translations` under topic_type. Maybe it's more clear if I actually fetch each translation here vs in the type and then pass that in context instead.